ci-docker-images/Dockerfile

```diff
@@ -12,9 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM oraclelinux:7.4
+FROM oraclelinux:7.5
 
-RUN yum install -y ca-certificates make openssl git jq && yum clean all
+RUN yum install --enablerepo=ol7_developer_EPEL -y unzip pwgen software-properties-common python git ca-certificates make openssl git jq && \
```

Comment:
Suggest breaking out each package being installed onto its own line and sorting alphabetically so it’s easy to see which packages are being added/removed in future.